JSDemos/Demos/CheckBox/Overview/React/App.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import { CheckBox } from 'devextreme-react/check-box';
 
+const checkedLabel = { 'aria-label': 'Checked' };
+const unCheckedLabel = { 'aria-label': 'Unchecked' };
```

Comment:
`un` is a prefix, not a separate word
```suggestion
const uncheckedLabel = { 'aria-label': 'Unchecked' };
```

Reply:
Fixed: [e74e9aa0894fdc88f313789f8799b626ce473904].